CHANGELOG.md

```diff
@@ -2,6 +2,19 @@ faucet-pipeline-core version history
 ====================================
 
 
+v1.7.0
+------
+
+_2022-02-09_
+
+notable changes for end users:
+
+* Node 14 and 17 are now officially supported as well
+* update to Browserslist 4.19.1
```

Comment:
```suggestion
no significant changes for end users
```

arguably none of this is significant; in the Before Times, we argued that burdening faucet users with low-level details was usually unnecessary and rarely helpful

so in fact, I'd consider this a patch update